test/unit/io_mouse.js

```diff
@@ -0,0 +1,49 @@
+var test = require('tap').test;
+var Mouse = require('../../src/io/mouse');
+var Runtime = require('../../src/engine/runtime');
+
+test('spec', function (t) {
+    var rt = new Runtime();
+    var m = new Mouse(rt);
+
+    t.type(m, 'object');
+    t.type(m.postData, 'function');
+    t.type(m.getX, 'function');
+    t.type(m.getY, 'function');
+    t.type(m.getIsDown, 'function');
+    t.end();
+});
+
+test('mouseUp', function (t) {
+    var rt = new Runtime();
+    var m = new Mouse(rt);
+
+    m.postData({
+        x: 1,
+        y: 10,
+        isDown: false,
+        canvasWidth: 480,
```

Comment:
note that we are planning to refactor what data needs to be posted after https://github.com/LLK/scratch-render/issues/29